nosnoc/model.py

```diff
@@ -32,6 +32,10 @@ class NosnocModel:
         different state equations with each boundary zone
     :param g_Stewart: List of stewart functions to define the regions (instead of S & c)
     :param u: controls
+    :param z: user defined algebraic state variables
+    :param z0: initial algebraic values
+    :param lbz: lower bounds on user algebraic variables
+    :param ubz: upper bounds on user algebraic variables
```

Comment:
Just thought if those should go to OCP, but you actually might need them for pure simulation.